rel/config.exs

```diff
@@ -53,4 +53,22 @@ release :golden_path do
       :runtime_tools
     ]
   )
+
+  set(
+    commands: [
+      migrate: "rel/commands/migrate.sh"
+    ]
+  )
+
+  set(
```

Comment:
This is basically a wonderful runtime Config Provider that allows us to use mix like terminology within a release, see here for more:

https://hexdocs.pm/distillery/config/runtime.html